linux/app-config.sh

```diff
@@ -1,44 +1,70 @@
 #! /bin/bash
 
 #
-#  gen-config.sh - Generates nrtest app configuration file for test executable
+#  app-config.sh - Generates nrtest app configuration file for test executable
 #
 #  Date Created: 11/15/2017
 #
-#  Author:       Michael E. Tryby
+#  Authors:      Michael E. Tryby
 #                US EPA - ORD/NRMRL
+#                
+#                Caleb A. Buahin
+#                Xylem Inc.
 #
+#  Requires:
+#    git
+#
+#  Environment Variables:
+#    PROJECT
+#    PLATFORM
+#                  
 #  Arguments:
 #    1 - absolute path to test executable
-#
-#  NOT IMPLEMENTED YET
-#    2 - test executable version number
-#    3 - build description
+#    2 - (SUT build it)
 #
 
 unameOut="$(uname -s)"
 case "${unameOut}" in
```

Comment:
Since we have separate scripts for Windows, Linux, and Darwin we can drop the case statement and simplify this.